.eslintrc.yaml

```diff
@@ -285,9 +289,10 @@ rules:
   react/prop-types: off  # We handle this better with types.
   react/require-default-props: off  # We handle this better with types.
 
-  # These two could be good to fix.
+  # These could be good to fix.
   react/prefer-stateless-function: off
   react/sort-comp: off
+  react/no-unstable-nested-components: off
```

Comment:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md

Hmm, yeah, the situation this looks for does sound bad.

Running it…
> ✖ 19 problems (19 errors, 0 warnings)

(in 7 different files)

Yeah, probably a followup commit (or a few commits) to fix these would be good.

I think most of them can just insert a `useMemo` and be done. In `Title.js` it's a bit more than that because they're conditional… hmm, no they're not, I think the rule is just getting a false positive. So an eslint-disable for the file should do it.

Reply:
Cool, yeah; I'll aim to come back to this.